.github/workflows/release-nightly.yaml

```diff
@@ -0,0 +1,45 @@
+name: nightly release
+
+on: push
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+        
+      - name: Run GoReleaser Nightly
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          distribution: goreleaser
+          version: nightly
+          args: release --rm-dist --config .goreleaser-nightly.yaml
```

Comment:
We don't want to release Github Release artifacts, we just want to publish docker artifacts. This makes the nightly release integration a bit non-trivial.

Could you please modify this release strategy to publish just the docker images and not run `goreleaser release` (which publishes the artifacts to Github Releases and publishes to Docker Hub).

Reply:
Resolved in [`0ec41b5`](https://github.com/openfga/openfga/pull/508/commits/0ec41b550b0c275a685c82edbcf4b6f836bd4db0)